content/blog/redfish-interoperability-profiles.md

```diff
@@ -0,0 +1,239 @@
+
+```shell
+# Profile Validator configuration file.
+#
+# Parameter description at:
+# https://github.com/DMTF/Redfish-Interop-Validator/blob/main/README.md 
+
+[Tool]
+Version = 2
+Copyright = Redfish DMTF (c) 2021
+verbose =
+
+[Host]
+ip = https://ilo-scott380g11-1
+username = username
+password = password
+description = iLO 6
+forceauth = False
+authtype = Session
+token =
+
+[Validator]
+# The following `tree` keyword tells the Validator
+# to crawl the Redfish tree starting at the following
+# starting point.
+#
+# An alternative is `single` to only validate the
+# starting point.
+payload =  tree /redfish/v1/Managers
+logdir = ./logs
+oemcheck = False
+online_profiles = False
+required_profiles_dir = 
+debugging = False
+collectionlimit =
+```
+
+The Validator outputs two files in the directory specified in the configuration file (`logdir`): a report in HTML format and a text file containing the different steps of the validation process.
+
+Figure 3 below is a portion of the Validator report showing three successful verifications. The first one (`Service level requirements`) requires the existence of the `ManagerCollection` and `Manager` end points. It has been automatically added by the Validator as a mandatory condition before proceeding the verification of the other requirements mentioned in the Profile.
+
+You can view as well on this figure the results of the two verifications required in the Profile.
+
+![Figure 3: Redfish conformance test report](/img/fig3-conformancetestreport.png "Redfish conformance test report")
+
+<figcaption>Figure 3: Redfish conformance test report</figcaption>
+
+## Leveraging Redfish Interoperability Profiles
+
+Although present for a long time (January 2018) and despite an introduction <a href="https://www.youtube.com/watch?v=iVAYSEPwmV8" target="_blank">video</a>, the Redfish Interoperability Profile specification is not very well known and it could be better leveraged by Redfish client programmers supervising heterogeneous data centers; this standard along with the Interoperability Validator highlight the differences between Redfish implementations and can help them to fast produce a more efficient code. If you already know a property is absent in a Redfish service, you can anticipate adapt your code early in the development process. 
+
+Then, Interoperability Profiles could also be leveraged by the purchasing departments of large organizations with several computer suppliers. By including such Profiles in their Requests for Proposal (RFPs) documents, they could make sure that the computers they purchase have a common baseline in terms of management features. Having a clear knowledge of such baseline is definite plus for security and supervising departments.
```

Comment:
Recommend rewording this first sentence to:
Interoperability profiles can also be leveraged by purchasing departments of large organizations who have several computer suppliers. 

In the next sentence, change "could" to "can" (it's a more powerful word):
By including such profiles in their Request for Proposal (RFPs) documents, they can make sure that the computers they purchase...

The last sentence also needs some slight modification:
Having a clear knowledge of this baseline is **a** definite plus for security and supervising departments.